docs/guides/adapting-new-services.md

```diff
@@ -2,6 +2,33 @@
 
 Please reach out in #harmony-service-providers (EOSDIS Slack) for additional guidance on any adaptation needs, and especially with any feedback that can help us improve.
 
+## Quick Start
+Fully setting up a service can be overwhelming for first time service providers. We now provide a script `bin/generate-new-service` to generate
+much of the scaffolding to get services ready for local integration testing with harmony quickly. The scaffolding provides the following:
+
+1. Updates to env-defaults files to add needed environment variables for running the service
+1. Updates to services.yml to fill in a new service definition to call the service
```

Comment:
```suggestion
1. Updates to services.yml to fill in a new service definition to call the service in our UAT environment.
```

Reply:
This quick start isn't meant for running in our UAT environment, it is just for running locally so mentioning calling the service in UAT would be confusing. But you are right though that I need to make it clear somewhere that these changes must be tested with collections in the CMR UAT environment (not production collections) - I'll clarify.